### ISSUE TITLE
Add in-program feedback tool

### ISSUE THREAD

**Issue by TianlanSha** [cairoshell/cairoshell#206]
Please consider adding a feedback tool inside the program. So that people testing it can provide feedback in a faster and easier way.

**Comment by josuave** [comment]
@dremin I have done some testing and can get the code to submit an issue to a repo. I want to see if we want this integrated right into cairo or want it to be an addin? Would you want a side discussion of this implementation?

**Comment by dremin** [comment]
@josuave I agree this would be a good candidate for an addin.

**Comment by dremin** [comment]
I think providing an interface for plugins to store data in the Cairo app settings would simplify the management of configuration files (I wouldn't mind moving app grabber and stacks data to the main settings file to be honest)
The end goal for plugins should be making the system flexible and robust enough that primary components of Cairo (app grabber, stacks, etc) would be able to become plugins, a good example of this that comes to mind is KDE Plasma.

**Comment by josuave** [comment]
I agree completely. The problem now comes with the UI. A pluggable Settings window. I'm not sure if I want to look at extending the building settings classes that .Net provides or move to another custom system or framework. I have been toying with the ApplicationSettingsBase class to see if I can get MEF to mix well with it. We may need to look at mixing a M V VM style system to add more flexibility.

**Comment by dremin** [comment]
We will implement this as a link since submitting an issue requires GitHub authentication.

**Comment by josuave** [comment]
Did we ever do this? Link for new issue: https://github.com/cairoshell/cairoshell/issues/new/choose
